app/models.py

```diff
@@ -1742,6 +1741,30 @@ def update_from_original(self, original):
         self.status = original.status
 
 
+# Indexes for notification_history and notifications to improve performance of fetch queries.
+Index(
```

Comment:
Minor: could we inline these with the model? I notice we have [a precedent for the inline approach already](https://github.com/alphagov/notifications-api/blob/916a1b55c5b0fb222d0accc9f5cbe98c786a0505/app/models.py#L831), which I prefer as it's easier to see which indices go with which tables.

I did [a quick test](https://github.com/alphagov/notifications-api/pull/new/sync-migrations-ben-test) to see if it would work for these ones as well, and it does (`flask db migrate` does nothing).